### PR TITLE
Upgrade Gradle to 4.0-milestone-1 and AGP to 3.0.0-alpha2.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

File: telecine/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ def versionBuild = 0 // bump for dogfood builds, public betas, etc.
 
 buildscript {
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.2.3'
+    classpath 'com.android.tools.build:gradle:3.0.0-alpha2'
```
Comment (NightlyNexus) [inline telecine/build.gradle]: this is a time bomb, though, right?

Comment (zach-klippenstein) [inline telecine/build.gradle]: True, feel free to decline.

Comment (JakeWharton) [inline telecine/build.gradle]: alpha3 is latest (although we'll get 4 tomorrow) and it has no time bomb

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-milestone-1-all.zip
```
Comment (JakeWharton) [inline gradle/wrapper/gradle-wrapper.properties]: RC2 is latest

File: telecine/build.gradle (hunk 1 above)
Comment (NightlyNexus) [inline telecine/build.gradle]: 🎉 Just on alpha4, or did the policy change?

Comment (JakeWharton) [inline telecine/build.gradle]: Some people got sad and complained when they were stuck on an alpha for months. Xav said it's function is relatively diminished now anyway, though, as it was just meant to keep people from being stuck on old 0.x and 1.x alphas years ago.

**Comment by JakeWharton** [comment]
Why hath CI abandoned me?

**Comment by JakeWharton** [comment]
Merged manually as 72c1b15dadf3373a39d915b50984bdda09669ee4

**Pull request closed without merging**